### PR TITLE
Update third-party-join.md

### PR DESCRIPTION
**Pull request by NancyForero**
1. Added the requirement to choose Microsoft Teams only mode in the console
2. Added the option to set Microsoft Teams only mode in the XML settings

### CODE REVIEW

File: Teams/rooms/third-party-join.md [hunk 1]
```diff
@@ -88,14 +88,22 @@ To configure Teams Rooms on Windows using the touchscreen console, do the follow
...
 1. On the Microsoft Teams Rooms console, select **More**.
 2. Select **Settings**, and then enter the device administrator username and password.
-3. Go to the **Meetings** tab and select a third-party meeting provider you wish to enable (e.g., **Webex**, **Zoom**, etc.).
-4. If you want to join meetings with the username and email address associated with the room mailbox, select **Join with room info**.
-5. If you want to join meetings with an alternate username and email address, select **Join with custom info** and enter username and email address you'd like to use.
-6. Select **Save and exit**. Your device will restart.
+3. Go to the **Account** tab and under **Supported meeting mode** choose **Microsoft Teams only**. 
+4. Go to the **Meetings** tab and select a third-party meeting provider you wish to enable (e.g., **Webex**, **Zoom**, etc.).
+5. If you want to join meetings with the username and email address associated with the room mailbox, select **Join with room info**.
+6. If you want to join meetings with an alternate username and email address, select **Join with custom info** and enter username and email address you'd like to use.
```
Comment (JohanFreelancer9) [inline Teams/rooms/third-party-join.md:94]:
```suggestion
6. If you want to join meetings with an alternate username and email address, select **Join with custom info** and enter the username and email address you'd like to use.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NancyForero, kindly commit the minor additional suggested change. Many thanks.

**Comment by NancyForero** [comment]
Yes. Looks good. Thanks.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by scanum** [comment]
Hi @NancyForero could you please confirm that the additional suggested changes have been committed?

**Pull request closed without merging**